.github/workflows/e2e-upgrade.yml

```diff
@@ -303,7 +487,7 @@ jobs:
 
       - name: Notify about failure
         if: |
-          failure() &&
+          always() &&
```

Comment:
not sure if this is for debugging, but we should only create failure tickets if the job actually failed